### ISSUE TITLE
Allow `buildScan.background` to launch external processes with configuration cache enabled

### ISSUE THREAD

**Issue by bigdaz** [gradle/gradle#20536]
This is required for the `common-custom-user-data-gradle-plugin`. See https://github.com/gradle/common-custom-user-data-gradle-plugin/issues/44

**Comment by mlopatkin** [comment]
BT-side changes are there: #20516. There is also #20566 that removes the restriction in 7.5, unless the build author explicitly enables `STABLE_CONFIGURATION_CACHE` feature.

**Comment by runningcode** [comment]
Is there a plan to remove this workaround? https://github.com/gradle/gradle/blob/master/subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/Workarounds.kt#L63

**Comment by mlopatkin** [comment]
> Is there a plan to remove this workaround? https://github.com/gradle/gradle/blob/master/subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/Workarounds.kt#L63

Yes, eventually. The `gradle/gradle` build has to update build scan plugin to a version that utilizes changes above, as our build enables all preview features, including STABLE_CONFIGURATION_CACHE.

**Comment by mlopatkin** [comment]
The Gradle-side part has landed, tracked in #22385. The adoption by the plugin is tracked in #22386.
